src/main/java/calculator/Opserator.java

```diff
@@ -0,0 +1,32 @@
+package calculator;
+
+public enum Opserator {
+
+    ADD{
+        @Override
+        public int getResult(CalculatorParamValue paramValue){
+            return paramValue.getFirstValue() + paramValue.getSecondValue();
+        }
+    },
+    DIV{
+        @Override
+        public int getResult(CalculatorParamValue paramValue){
+            return paramValue.getFirstValue() / paramValue.getSecondValue();
+        }
+    },
+    MUL{
+        @Override
+        public int getResult(CalculatorParamValue paramValue){
+            return paramValue.getFirstValue() * paramValue.getSecondValue();
+        }
+    },
+    SUB{
+        @Override
+        public int getResult(CalculatorParamValue paramValue){
+            return paramValue.getFirstValue() - paramValue.getSecondValue();
+        }
+    }
+    ;
```

Comment:
다음과 같이 변경되면 연산자에 대한 정보와 로직이 해당 객체에 한곳에 모여 응집도가 높아질 것 같아요!
해당 피드백과 같이 고민해보시면 좋을 것 같아요 :)

```suggestion

    ADD("+") {
        @Override
        public int getResult(CalculatorParamValue paramValue){
            return paramValue.getFirstValue() + paramValue.getSecondValue();
        }
    },
    SUB("-") {
        @Override
        public int getResult(CalculatorParamValue paramValue){
            return paramValue.getFirstValue() - paramValue.getSecondValue();
        }
    },
    MUL("*") {
        @Override
        public int getResult(CalculatorParamValue paramValue){
            return paramValue.getFirstValue() * paramValue.getSecondValue();
        }
    },
    DIV("/") {
        @Override
        public int getResult(CalculatorParamValue paramValue){
            return paramValue.getFirstValue() / paramValue.getSecondValue();
        }
    };

    private final String value;

    Operators(String value) {
        this.value = value;
    }
```